content/vault/v1.21.x (rc)/content/partials/alerts/secret-engine-tune-gui-support.mdx

```diff
@@ -0,0 +1,8 @@
+<Note title="Secret Engine Tune GUI Support">
```

Comment:
```suggestion
<Note title="Vault GUI plugin tuning support">
```
Suggest generalizing the title in case we expand the tuning to other things

Reply:
good call, thank you! - done